Assets/_BForBoss/_Entities/Scripts/LifeCycle.cs

```diff
@@ -31,12 +31,17 @@ public class LifeCycle : ILifeCycle
         public float MaxHealth => _maxHealth;
         public float CurrentHealth => _currentHealth;
 
-        public LifeCycle(IHealth health)
+        public LifeCycle(float health = 100f)
         {
-            _maxHealth = _currentHealth = health.Health;
+            _maxHealth = _currentHealth = health;
             IsAlive = true;
         }
 
+        public void SetHealth(float health)
+        {
+            _maxHealth = _currentHealth = health;
+        }
```

Comment:
This is really the only way i can think of injecting the health amount later on in the Initialize cycle